lib/GenomeFileUtil/core/GenbankToGenome.py

```diff
@@ -639,22 +624,49 @@ def _get_seq(self, feat, contig):
                                [part.start:part.end].reverse_complement()))
         return "".join(seq)
 
+    def _create_ontology_event(self,ontology_type):
```

Comment:
Can we move this and the identical version in GFF uploader into the utils so they are not duplicated? (If we chose to support a new ontology, only want to have to make a change in one place)